lib/Serializer/XmlEncoder.php

```diff
@@ -202,6 +202,13 @@ private function createParameter($parentEl, $name, $value)
             return $parameterEl;
         }
 
+        if (is_null($value)) {
+            $parameterEl->setAttribute('nillable', 'true');
+            $parameterEl->setAttribute('value', '');
+
+            return $parameterEl;
+        }
```

Comment:
May need to modify the `XmlDecoder` also

Reply:
I've had a quick look and found no impact. but i will review it ASAP